Event_spec.ts

```diff
@@ -203,20 +210,25 @@ type EventType =
     "berth.ata.portAuthority" |
     "berth.ata.terminal" |
     "berth.ata.vessel" |
+    "berth.ata.carrier" |
     "berth.atd.portAuthority" |
     "berth.atd.terminal" |
     "berth.atd.vessel" |
+    "berth.ata.carrier" |
     "berth.cancel.agent" |
     "berth.cancel.portAuthority" |
     "berth.cancel.terminal" |
+    "berth.cancel.carrier" |
```

Comment:
Do we need this and `port.cancel.carrier`?

Reply:
Good question, I've only done both of them because our domain contained both. I guess it's possible they would cancel a berth visit but I'd find that highly unlikely